### DISCUSSION TITLE
cmp suddenly show me an error as soon as I navigate in the suggestions

### DISCUSSION THREAD

**Discussion by kimagin** [LazyVim/LazyVim#4064] [Q&A]
A few hours ago, suddenly cmp throws the following error, as soon as I hit <C-n> or <C-p> :

E5108: Error executing lua: .../nvim/lazy/nvim-cmp/lua/cmp/view/custom_entries_view.lua:328: attempt to perform arithmetic on local 'cursor' (a nil value)
stack traceback:
	.../nvim/lazy/nvim-cmp/lua/cmp/view/custom_entries_view.lua:328: in function 'select_next_item'
	...ination/.local/share/nvim/lazy/nvim-cmp/lua/cmp/view.lua:224: in function 'select_next_item'
	...ination/.local/share/nvim/lazy/nvim-cmp/lua/cmp/init.lua:146: in function 'select_next_item'
	...ocal/share/nvim/lazy/nvim-cmp/lua/cmp/config/mapping.lua:202: in function 'on_keymap'
	...ination/.local/share/nvim/lazy/nvim-cmp/lua/cmp/core.lua:145: in function 'callback'
	....local/share/nvim/lazy/nvim-cmp/lua/cmp/utils/keymap.lua:133: in function <....local/share/nvim/lazy/nvim-cmp/lua/cmp/utils/keymap.lua:127>
	
Same behaviour is happening when I am using up and down arrows.
	
Things that I've tried in order to solve the issue:
- removing .local/state/nvim
- removing .local/share/nvim
- removing .cache/nvim
- removing .config/nvim and installing a fresh version of Lazyvim
- removing every file related to Neovim and re installing nvim v0.10.0

The issue is happening on a macOS system.
I will appreciate any help to resolve the issue.

**Comment by kimagin** [comment]
I am closing this issue as the problem got solved by it self. Probably it was a temporary bug.

**Reply by iamxiaojianzheng** [reply]
I had this problem too, and I don't know why.

**Reply by iamxiaojianzheng** [reply]
I reinstall `nvim-cmp` to solve the problem.
